common/scripts/x86_64/Ubuntu_16.04/_helpers.sh

```diff
@@ -13,6 +13,8 @@ export ADMIRAL_IMAGE="u16admiral"
 __check_dependencies() {
   __process_marker "Checking dependencies"
 
+  apt-get update
```

Comment:
lets add a `-y` just to be safe

Reply:
i dont think this is required for update